utils/build_swift/defaults.py

```diff
@@ -0,0 +1,52 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+"""
+Default option value definitions.
+"""
+
+__all__ = [
+    # Command line configuarable
+    'SWIFT_USER_VISIBLE_VERSION',
+    'CLANG_USER_VISIBLE_VERSION',
+    'SWIFT_ANALYZE_CODE_COVERAGE',
+    'DARWIN_XCRUN_TOOLCHAIN',
+    'DARWIN_DEPLOYMENT_VERSION_OSX',
+    'DARWIN_DEPLOYMENT_VERSION_IOS',
+    'DARWIN_DEPLOYMENT_VERSION_TVOS',
+    'DARWIN_DEPLOYMENT_VERSION_WATCHOS',
+    'UNIX_INSTALL_PREFIX',
+    'DARWIN_INSTALL_PREFIX',
+
+    # Constants
+]
+
+# Options that can be "configured" by command line options
+
+BUILD_VARIANT = 'Debug'
+CMAKE_GENERATOR = 'Ninja'
+
+COMPILER_VENDOR = 'none'
+SWIFT_USER_VISIBLE_VERSION = '4.1'
+CLANG_USER_VISIBLE_VERSION = '5.0.0'
+SWIFT_ANALYZE_CODE_COVERAGE = 'false'
+
+DARWIN_XCRUN_TOOLCHAIN = 'default'
+DARWIN_DEPLOYMENT_VERSION_OSX = '10.9'
+DARWIN_DEPLOYMENT_VERSION_IOS = '7.0'
+DARWIN_DEPLOYMENT_VERSION_TVOS = '9.0'
+DARWIN_DEPLOYMENT_VERSION_WATCHOS = '2.0'
+
+UNIX_INSTALL_PREFIX = '/usr'
+DARWIN_INSTALL_PREFIX = ('/Applications/Xcode.app/Contents/Developer/'
```

Comment:
I noticed that some of the options here are not used below. Why are they here? Specifically UNIX_INSTALL_PREFIX or am I missing something?

Reply:
They will be used very soon. I just didn't want to add more complexity in #11827.